src/main/java/sirius/web/util/SystemHelper.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.util;
+
+import sirius.kernel.Setup;
+import sirius.kernel.Sirius;
+import sirius.kernel.di.std.Register;
+import sirius.web.security.HelperFactory;
+import sirius.web.security.ScopeInfo;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Provides static system functionality and system environment detection based on {@link sirius.kernel.Setup.Mode} as
+ * helper for in-template usage.
+ */
+public class SystemHelper {
```

Comment:
we could also encapsulate all this in a isProd() / isStaging() / isDev() macro which is probably easier to use anyway... (?)

Reply:
This will be rebuilt into a macro as suggested.